src/main/resources/templates/db_scripts/schema.json

```diff
@@ -24,6 +24,7 @@
     {
       "tableName": "fund",
       "generateId": true,
+      "customSnippetPath": "fund_table.sql",
```

Comment:
Please move unique index definition to `schema.json` (sample can be found in [mod-invoice-storage](https://github.com/folio-org/mod-invoice-storage/blob/master/src/main/resources/templates/db_scripts/schema.json#L25)). Please refer to [RMB documentation](https://github.com/folio-org/raml-module-builder#the-post-tenant-api)

Reply:
updated to use RMB features